api/ruxos_posix_api/src/lib.rs

```diff
@@ -45,7 +45,8 @@ pub mod config {
 #[allow(dead_code, non_snake_case, non_camel_case_types, non_upper_case_globals, clippy::upper_case_acronyms, missing_docs)]
 pub mod ctypes;
 
-pub use imp::io::{sys_read, sys_readv, sys_write, sys_writev};
+pub use imp::getrandom::{rand, random, srand, sys_getrandom};
```

Comment:
Only pub systems calls instead of APIs. Currently, let's regard `rand, random, srand` as syscalls for ruxlibc. Change into `sys_rand, sys_random, sys_srand`.